### PR TITLE
DOMA-2500 Fix updating property when changing string after accepting …

### PR DESCRIPTION
**Pull request by VKislov**
…property

### CODE REVIEW

File: apps/condo/domains/common/components/BaseSearchInput/index.tsx [hunk 1]
```diff
@@ -138,6 +138,14 @@ export const BaseSearchInput = <S extends string>(props: ISearchInput<S>) => {
         onChange(value, options)
     }, [onChange])
 
+    useEffect(()=>{
+        if (selected !== null && (selected !== searchValue) ) {
```
Comment (akarjakin) [inline apps/condo/domains/common/components/BaseSearchInput/index.tsx:142]: `if (selected !== null && selected !== searchValue) {`

File: apps/condo/domains/common/components/BaseSearchInput/index.tsx [hunk 2]
```diff
@@ -138,6 +138,14 @@ export const BaseSearchInput = <S extends string>(props: ISearchInput<S>) => {
...
+            setSelected(null)
+            setSearchValue(undefined)
+            props.onClear()
+        }
+    }, [searchValue])
```
Comment (akarjakin) [inline apps/condo/domains/common/components/BaseSearchInput/index.tsx:147]: `[searchValue, props.onClear]`

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/common/components/BaseSearchInput/index.tsx [hunk 3]
```diff
@@ -138,6 +138,14 @@ export const BaseSearchInput = <S extends string>(props: ISearchInput<S>) => {
         onChange(value, options)
     }, [onChange])
 
+    useEffect(()=>{
```
Comment (pahaz) [inline apps/condo/domains/common/components/BaseSearchInput/index.tsx:141]: Write a comment.

Comment (VKislov) [reply apps/condo/domains/common/components/BaseSearchInput/index.tsx:141]: Wrote in new pr. https://github.com/open-condo-software/condo/pull/1514

**Pull request merged**